spec/Body/Body.vspec

```diff
@@ -161,11 +161,57 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights.
+#include StaticLights.vspec Lights.Beam
+
+Lights.Running:
+  type: branch
+  description: Running lights.
+#include StaticLights.vspec Lights.Running
+
+Lights.Backup:
+  type: branch
+  description: Backup lights.
+#include StaticLights.vspec Lights.Backup
+
+Lights.Parking:
+  type: branch
+  description: Parking lights.
+#include StaticLights.vspec Lights.Parking
+
+Lights.Fog:
+  type: branch
+  instances: ["Rear","Front"]
+  description: Fog lights.
+#include StaticLights.vspec Lights.Fog
+
+Lights.LicensePlate:
+  type: branch
+  instances: ["Rear","Front"]
```

Comment:
Is there any country that mandates that front license plate is illuminated? And/Or any vehicles that support it?

Reply:
@erikbosch Does it really matter if the license plate light is mandatory or not in a particular country? I would say not. As far as I understand, the idea is that VSS provides the means to name and access the data point of interest. In this case, it would be sufficient that there exist one car that has this type of light to model it.

Reply:
Agree, but do we know of any vehicle supporting it today. Just want to make sure that we do not add an instance that likely not will be used.